charts/alertmanager/templates/_helpers.tpl

```diff
@@ -41,6 +41,9 @@ helm.sh/chart: {{ include "alertmanager.chart" . }}
 app.kubernetes.io/version: {{ . | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- with .Values.commonMetaLabels}}
```

Comment:
please rename to extraLabels:
```suggestion
{{- with .Values.extraLabels }}
```

Please also add "extraLabels" to values.yaml.